src/interpret/lianapy/config.vsh.yaml

```diff
@@ -0,0 +1,84 @@
+functionality:
+  name: lianapy
+  namespace: "interpret"
+  description: "Performs LIANA integration based as described in https://github.com/saezlab/liana-py"
+  arguments:
+    - name: "--input"
+      alternatives: ["-i"]
+      type: file
+      description: Input h5mu file
+      direction: input
+      required: true
+    - name: "--output"
+      alternatives: ["-o"]
+      type: file
+      description: Output h5mu file.
+      direction: output
+      required: true
+    - name: "--compression"
```

Comment:
Could you use `choices:` here to describe the different values for this parameter? Like `choices: ["gzip", "lzf"]`?

Reply:
Also, could you rename this to `output_compression`, to align with the other components?